dot-atom/keymap.cson

```diff
@@ -2,7 +2,8 @@
 #
 # Atom keymaps work similarly to style sheets. Just as style sheets use
 # selectors to apply styles to elements, Atom keymaps use selectors to associate
-# keystrokes with events in specific contexts.
+# keystrokes with events in specific contexts. (Unlike style sheets however,
+# each selector can only be declared once.)
```

Comment:
I think we can drop the parens here. 

Also, it might be helpful to give an example of the error, and then how the error should be corrected (to make things perfectly clear for users who just copy/paste examples). For example:

```
# This is not correct
#
# 'atom-text-editor':
#   'enter': 'editor:newline'
# 'atom-text-editor':
#   'end': 'editor:move-to-end-of-screen-line'
#
# This is correct:
#
# 'atom-text-editor':
#   'enter': 'editor:newline'
#   'end': 'editor:move-to-end-of-screen-line'
```

Another approach would be to fix https://github.com/atom/docs/issues/43 :memo:, and then link to those docs with a big WARNING. I think I like that approach a bit more.